src/inverseRobot.js

```diff
@@ -6,8 +6,22 @@
  * @return {object}
 */
 
+// const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+// const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
+// inverseRobot(robert) === null
+// inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
 function inverseRobot(robot) {
-  // write code here
-}
+  const tobor = {};
+
+  Object.keys(robot).forEach(key => {
+    // console.log('key =', key, ', robot[key] =', robot[key]);
+    if (!tobor[robot[key]]) {
+      tobor[robot[key]] = key;
+    }
```

Comment:
you repeat robot[key] 2 times here, better create variable with this value

Reply:
const created